static/app/components/searchSyntax/renderer.tsx

```diff
@@ -371,9 +369,13 @@ const Unit = styled('span')`
   color: ${p => p.theme.green400};
 `;
 
-const LogicBoolean = styled('span')`
+const LogicBoolean = styled('span')<{invalid: boolean}>`
   font-weight: bold;
   color: ${p => p.theme.gray300};
+  ${p =>
+    p.invalid &&
+    `
+    color: ${p.theme.red400}`}
 `;
```

Comment:
Why did this end up broken across lines?
```suggestion
  ${p => p.invalid && `color: ${p.theme.red400};`}
`;
```